msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -105,9 +107,11 @@ internal ITelemetryReceiver TelemetryReceiver
             CryptographyManager = PlatformProxyFactory.GetPlatformProxy().CryptographyManager;
             WsTrustWebRequestManager = new WsTrustWebRequestManager(HttpManager);
             TelemetryManager = telemetryManager ?? new TelemetryManager(Telemetry.GetInstance());
+            ValidatedAuthoritiesCache = new ValidatedAuthoritiesCache();
```

Comment:
>ValidatedAuthoritiesCache = new ValidatedAuthoritiesCache(); [](start = 12, length = 60)

as discussed and to Bogdan's point, it might be useful to identify a way for the user to enable sharing. 
However we will in that case though also have to ensure we do cache refresh at the appropriate time (if people are keeping e.g. servers running for a very long time with out flushing memory/restaring/etc.)